src/types/BaseField.ts

```diff
@@ -117,19 +113,62 @@ const writableBaseFieldSchema: JSONSchemaType<WritableBaseField> = {
 	required: [
 		'label',
 		'description',
-		'shortCode',
 		'dataType',
 		'scope',
 		'valueRelevanceHours',
 	],
 };
 
+const internallyWritableBaseFieldSchema: JSONSchemaType<InternallyWritableBaseField> =
```

Comment:
I don't think this (or the related type guard) is actually used anywhere, right?  So we wouldn't need to make it.